Gems/Multiplayer/Code/Tests/MultiplayerSystemTests.cpp

```diff
@@ -77,10 +80,18 @@ namespace Multiplayer
                 });
             m_mpComponent->AddEndpointDisconnectedHandler(m_endpointDisconnectedHandler);
             m_mpComponent->Activate();
+
+            m_systemEntity = AZStd::make_unique<AZ::Entity>();
+            m_systemEntity->CreateComponent<AZ::JobManagerComponent>(); // Needed by Job system when @sv_multithreadedConnectionUpdates is on.
+            m_systemEntity->Init();
+            m_systemEntity->Activate();
```

Comment:
Well this isn't really the SystemEntity since it doesn't have EntityID 0.
This is just an entity that contains the JobManagerComponent

Reply:
Last I looked, a system entity had an ID of 0. Has that changed?

Reply:
The System Entity does have an entity ID of zero, but the Entity class default constructor does [NOT generate](https://github.com/o3de/o3de/blob/development/Code/Framework/AzCore/AzCore/Component/Entity.cpp#L82-L85) an Entity with ID 0.
It generates a psuedo random ID by combining the process ID + the machine ID

Reply:
Oh, good catch! Fixed.